com.unity.shadergraph/Editor/Data/Graphs/VirtualTextureShaderProperty.cs

```diff
@@ -74,7 +74,7 @@ internal override void ForeachHLSLProperty(Action<HLSLProperty> action)
             int numLayers = value.layers.Count;
             if (numLayers > 0)
             {
-                action(new HLSLProperty(HLSLType._CUSTOM, referenceName, HLSLDeclaration.UnityPerMaterial, concretePrecision)
+                action(new HLSLProperty(HLSLType._CUSTOM, referenceName + "_CBDecl", HLSLDeclaration.UnityPerMaterial, concretePrecision)
```

Comment:
the code that looks for duplicate HLSLProperty got caught up on the fact that VT uses the same name here -- it's not necessary since those names are just placeholders to inject custom VT declarations, so I made them unique.

Reply:
Magic string suffix is a little awkward, but I wouldn't do it any other way.